components/ModDashboardHealthProfessionalCard.vue

```diff
@@ -0,0 +1,137 @@
+<template>
+    <div class="w-96 my-1">
+        <div
+            class="flex justify-between items-center rounded-lg p-1 border-2 border-primary"
+        >
+            <div class="w-16 self-start">
+                <SVGProfileIcon
+                    role="img"
+                    alt="profile icon"
+                    title="profile icon"
+                    class="profile-icon stroke-primary w-16 h-16 stroke-1 inline mx-1 self-start"
+                />
+            </div>
+            <div v-if="moderationScreenStore.activeScreen === ModerationScreen.EditFacility">
+                <div class="flex flex-col h-full w-64 pl-1 mb-1">
+                    <div class="flex font-bold pt-2">
+                        <span>{{ healthcareProfessional.names[0].lastName }}</span>
```

Comment:
I did it where it defaults to 0 index since I believe at this current moment we don't require an English name. It can be localized to any language when adding to the database